src/lenskit/data/relationships.py

```diff
@@ -103,7 +121,10 @@ def attribute_names(self) -> list[str]:
 
     def count(self):
         if "count" in self._table.column_names:  # pragma: nocover
-            raise NotImplementedError()
+            count_column = self._table.column("count").combine_chunks().slice(0)
+            count_column.fill_null(pa.scalar(1, type=pa.int8()))
```

Comment:
If this doens't work without `combine_chunks`, use `pc.fill_null` instead. With `pc.fill_null`, possibly also `column.fill_null`, we can also just pass `1` without passing it through `pa.scalar` (the type checker doesn't like it but it works, another problem with the PyArrow type stubs).